distributed/utils.py

```diff
@@ -359,12 +359,17 @@ def convert(task):
 
 import logging
 from .compatibility import logging_names
-logging.basicConfig(format='%(name)s - %(levelname)s - %(message)s',
-                    level=logging.INFO)
+
+handler = logging.StreamHandler(sys.stderr)
+handler.setLevel(logging.INFO)
+fmt = logging.Formatter('%(name)s - %(levelname)s - %(message)s')
+handler.setFormatter(fmt)
 
 for name, level in config.get('logging', {}).items():
```

Comment:
By default, this won't configure any logger, though? You should add a default configuration for the "distributed" logger, it will be inherited by any child loggers (any "distributed.FOO.BAR").